.github/dependabot.yml

```diff
@@ -7,3 +7,12 @@ updates:
   commit-message:
     prefix: fix
     include: scope
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
+  labels:
+  - kind/deps
```

Comment:
Let us instead add the `area/dependency` label ([ref](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#area/dependency)) conforming to k8s standards. Alongside that, a second label `ok-to-test` would allow Prow to run tests prior to review automatically.
```suggestion
  - "area/dependency"
  - "ok-to-test"
```